staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go

```diff
@@ -319,6 +319,10 @@ func (o LogsOptions) RunLogs() error {
 		return err
 	}
 
+	if len(requests) == 0 {
+		return fmt.Errorf("No resources found in %s namespace.", o.Namespace)
```

Comment:
errors should start with lowercase and not end with a period.

Reply:
@neolit123 

Thank you for your review! I have fixed it now.